Project_First/ViewController.swift

```diff
@@ -8,45 +8,46 @@
 
 import UIKit
 
-class ListController: UICollectionViewController {
+final class SearchViewController: UIViewController {
     
-    var repositaries = ["Alamofire","Realm","KingFisher","SnapKit"]
     private enum Section: CaseIterable {
-    case main
+        case main
     }
-    private lazy var dataSource: UICollectionViewDiffableDataSource<Section, [String]> = {
+    
+    private var repositories = ["Alamofire", "Realm", "KingFisher", "SnapKit"]
+    
+    private lazy var collectionView: UICollectionView = {
+        let config = UICollectionLayoutListConfiguration(appearance: .insetGrouped)
+        let layout = UICollectionViewCompositionalLayout.list(using: config)
+
+        let collectionView = UICollectionView(
+            frame: view.bounds,
+            collectionViewLayout: layout
+        )
+        collectionView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+
+        return collectionView
+    }()
+    
+    private lazy var dataSource: UICollectionViewDiffableDataSource<Section, String> = {
         let cellRegistration = UICollectionView.CellRegistration<UICollectionViewListCell, [String]>
-          { cell, indexPath, repositary in
+          { cell, indexPath, repository in
```

Comment:
Мы не используем переменную `indexPath`, поэтому надо заменить её `_`. Так проще понимать код, потому что не придется глазами искать, где используется эта переменная, получится так:
```suggestion
          { cell, _, repository in
```